python_modules/dagster-graphql/dagster_graphql/schema/logs/events.py

```diff
@@ -309,14 +318,25 @@ class Meta:
     externalUrl = graphene.String()
     externalStdoutUrl = graphene.String()
     externalStderrUrl = graphene.String()
-    logManagerMetadata = graphene.String()  # TODO - replace with union?
+    logManagerMetadataRaw = graphene.JSONString()
```

Comment:
not sure if we've used `JSONString` before - not sure what it does

Reply:
it's actually weird that this works - probably doesn't break only because my query avoids this field. 

this isn't intended to be merged as this, just a temp hack I did to pass the data to the resolver below while figuring how I should actually hook this metadata (see questions in description).